Sources/MapboxMaps/Style/Generated/Layers/SkyLayer.swift

```diff
@@ -2,6 +2,7 @@
 import Foundation
 
 /// A spherical dome around the map that is always rendered behind all other layers.
+/// Warning: As of v10.6.0, atmosphere layer is the preferred method for atmospheric styling. Sky layer is not supported by the globe projection, and will be phased out in future major release.
```

Comment:
nit: we can link to the Atmosphere definition for better clarity here.
```suggestion
/// Warning: As of v10.6.0, ``Atmosphere`` layer is the preferred method for atmospheric styling. Sky layer is not supported by the globe projection, and will be phased out in future major release.
```

Reply:
@maios will it add a link or should I add some import explicitly?